frontend/src/templates/Field/Checkbox/CheckboxField.tsx

```diff
@@ -84,6 +84,7 @@ export const CheckboxField = ({
           colorScheme={fieldColorScheme}
           key={idx}
           value={o}
+          defaultValue={[]}
```

Comment:
the `defaultValue` for this looks wrong. This is value for a single checkbox. Should it be `defaultValue=''` instead?

Reply:
fixed